### ISSUE TITLE
(Future request) Time-marked messages

### ISSUE THREAD

**Issue by hemee** [ChatSecure/ChatSecure-iOS#785]
I just thought it would be nice if I see the time when I exactly sent or received a message near (under or from the side) a message itself. It'd make chat screen more convenient to use, I guess.

**Comment by paskalito** [comment]
i'm not sure but i guess you can tap on the message and then it will show the time.. :)

**Comment by hemee** [comment]
@paskalito No, it doesn't work. Imessage allows to view the time by swiping a message, but it's not the case here.
